lib/api/objectPut.js

```diff
@@ -13,23 +14,29 @@ import { logger } from '../utilities/logger';
 import { pushMetric } from '../utapi/utilities';
 import kms from '../kms/wrapper';
 import removeAWSChunked from './apiUtils/object/removeAWSChunked';
+import { metadataValidateBucketAndObj } from
+    './apiUtils/authorization/validation';
+import metadata from '../metadata/wrapper';
+
+const versionIdUtils = versioning.VersionID;
+
 
 function _storeInMDandDeleteData(bucketName, dataGetInfo, cipherBundle,
     metadataStoreParams, dataToDelete, deleteLog, callback) {
     services.metadataStoreObject(bucketName, dataGetInfo,
-        cipherBundle, metadataStoreParams, (err, contentMD5) => {
+        cipherBundle, metadataStoreParams, (err, res) => {
```

Comment:
let's call this something more descriptive than res especially since it's not the actual response object